Makefile

```diff
@@ -5,9 +5,9 @@ export GO111MODULE=on
 VERSION = $(shell git describe --tags --dirty --always)
 ECR_VERSION = $(VERSION)-$(ZQD_K8S_USER)
 LDFLAGS = -s -X github.com/brimsec/zq/cli.Version=$(VERSION)
-ZEEKTAG = v3.2.1-brim4
+ZEEKTAG = $(shell python3 -c 'import json,sys;print(json.load(open("package.json","rt"))["brimdependencies"]["zeektag"])')
```

Comment:
At first I used a simpler-looking `jq` command line to do this. However, it looks like `jq` is only used by the CI, whereas Python 3 is called out as a base testing dependency in [CONTRIBUTING.md](https://github.com/brimsec/zq/blob/master/CONTRIBUTING.md), so I'm opting for this slightly uglier approach.